docs/sources/get-started/overview.md

```diff
@@ -1,76 +1,52 @@
 ---
-menuTitle: Overview
+menuTitle: Loki overview
 title: Loki overview
 description: Loki product overview and features.
 weight: 200
 aliases:
     - ../overview/
     - ../fundamentals/overview/
 ---
-# Loki overview
-
-Grafana Loki is a log aggregation tool,
-and it is the core of a fully-featured logging stack.
-
-Loki is a datastore optimized for efficiently holding log data.
-The efficient indexing of log data
-distinguishes Loki from other logging systems.
-Unlike other logging systems, a Loki index is built from labels,
-leaving the original log message unindexed.
-
-![Loki overview](../loki-overview-1.png "Loki overview")
 
-An agent (also called a client) acquires logs,
-turns the logs into streams,
-and pushes the streams to Loki through an HTTP API.
-The Promtail agent is designed for Loki installations,
-but many other [Agents]({{< relref "../send-data" >}}) seamlessly integrate with Loki.
-
-![Loki agent interaction](../loki-overview-2.png "Loki agent interaction") 
+# Loki overview
 
-Loki indexes streams.
-Each stream identifies a set of logs associated with a unique set of labels.
-A quality set of labels is key to the creation of an index that is both compact
-and allows for efficient query execution.
+Loki is a horizontally-scalable, highly-available, multi-tenant log aggregation system inspired by [Prometheus](https://prometheus.io/). Loki differs from Prometheus by focusing on logs instead of metrics, and collecting logs via push, instead of pull.
 
-[LogQL]({{< relref "../query" >}}) is the query language for Loki.
+Loki is designed to be very cost effective and highly scalable. Unlike other logging systems, Loki does not index the contents of the logs, but only indexes metadata about your logs as a set of labels for each log stream. 
 
-## Loki features
+A log stream is a set of logs which share the same labels. Labels help Loki to find a log stream within your data store, so having a quality set of labels is key to efficient query execution.
 
--  **Efficient memory usage for indexing the logs**
+Log data is then compressed and stored in chunks in an object store such as Amazon S3 or Google Cloud Storage, or even, for development or proof of concept, on the filesystem. A small index and highly compressed chunks simplifies the operation and significantly lowers the cost of Loki.
```

Comment:
Here you use the full name of the storage services.

Reply:
Ah, that's where I updated it!